### DISCUSSION TITLE
Does this project have enough maintainer help?

### DISCUSSION THREAD

**Discussion by ourchitectureio** [bigskysoftware/htmx#1817] [Q&A]
I noticed that this project seems active, with recent commits. But, I also noticed that there are a lot of pull requests and that they stretch to a year ago. Does this project have enough maintainer help?

**Comment by alexpetros** [comment]
Can't speak to "enough" one way or another but two months ago we had about 60 open PRs and right now we have 36, so the trajectory is pretty good on that front. I expect that within a couple months we'll be in the 10-15 range, which is very healthy for a project of this size.
